e2e-tests/gef/cypress/e2e/facility/unissued-facilities/unissued-facilities-change-all-AIN.spec.js

```diff
@@ -255,11 +255,6 @@ context('Unissued Facilities AIN - change all to issued from unissued table', ()
       aboutFacilityUnissued.issueDateError().contains('The year for the issue date must include 4 numbers');
       aboutFacilityUnissued.coverStartDateError().contains('The year for the cover start date must include 4 numbers');
       aboutFacilityUnissued.coverEndDateError().contains('The year for the cover end date must include 4 numbers');
-
-      if (facilityEndDateEnabled) {
```

Comment:
Because the facility should have a pre-existing answer to this section & you cant uncheck the radio buttons it *shouldnt* be possible to get this error message to appear.

The logic is all commonised though & tested in `about-facility.spec` so this should be all good.

Hope that justifies removing test coverage